test/test_models.py

```diff
@@ -325,7 +325,16 @@ def test_fasterrcnn_double(self):
         self.assertTrue("labels" in out[0])
 
     def test_googlenet_eval(self):
-        m = torch.jit.script(models.googlenet(pretrained=True).eval())
+        # replacement for models.googlenet(pretrained=True) that does not download weights
+        kwargs = {}
```

Comment:
Seems like `kwargs` is not used ?

Reply:
thanks for catching this. Should be resolved.